packages/terra-site/webpack.prod.config.js

```diff
@@ -16,9 +17,13 @@ config.plugins.push(new webpack.DefinePlugin({
 // Clean build before running
 config.plugins.push(new CleanPlugin('build', { exclude: ['stats.json'] }));
 
+// Minify css and js
+config.plugins.push(new webpack.LoaderOptionsPlugin({ minimize: true }));
+
```

Comment:
Yeah, let's tackle adding UglifyJsPlugin or bablili in another PR. I agree with the concerns you listed @windse7en. I'll log an issue to explore this in a separate PR.